<?xml version="1.0" encoding="utf-8"?>

<launch>

    <!-- # arm controller -->
    <node pkg="my_robot" type="arm_servo.py" name="arm_servo" output="screen">
        <rosparam file="$(find my_robot)/config/arm_servo.yaml" command="load" />
    </node>

</launch>
